src/Employer/Employer.Web/Orchestrators/VacancyPreviewOrchestrator.cs

```diff
@@ -115,22 +119,31 @@ private async Task<SubmitVacancyResponse> SubmitActionAsync(Vacancy vacancy, Vac
             return response;
         }
 
-        private void SyncErrorsAndModel(IList<EntityValidationError> errors)
+        private void FlattenErrors(IList<EntityValidationError> errors)
         {
-            //Flatten Qualification errors to its ViewModel parent instead. 'Qualifications[1].Grade' > 'Qualifications'
+            //Flatten Qualification errors to its ViewModel parent instead. 'Qualifications[1].Grade' becomes 'Qualifications'
             foreach (var error in errors.Where(e => e.PropertyName.StartsWith(nameof(Vacancy.Qualifications))))
             {
                 error.PropertyName = nameof(VacancyPreviewViewModel.Qualifications);
             }
         }
 
+        private EntityValidationResult GetSoftValidationErrors(Vacancy vacancy)
+        {
+            var result = _vacancyClient.Validate(vacancy, SoftValidationRules);
+            FlattenErrors(result.Errors);
```

Comment:
Do we need to run FlattenErrors here when you are only validating on MinimumWage and Flatten Errors is really only specific for Qualifications errors?

Reply:
This was my effort to keep the mapping of vacancy to ModelState the same between the Get and the POST.  I'll move this into a common mapping function to be more explicit. 👍